core/src/software/aws/toolkits/core/credentials/StaticCredentialsToolkitCredentialsProvider.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.core.credentials
+
+import software.amazon.awssdk.auth.credentials.AwsCredentials
+
+class StaticCredentialsToolkitCredentialsProvider(private val awsCredentials: AwsCredentials) : ToolkitCredentialsProvider() {
```

Comment:
Does `StaticCredentialsProvider` in the SDK fulfill the same purpose as this class?

Reply:
@abrooksv to answer this--I didn't touch any of the credential code.

Reply:
It's required since I re-used ToolkitClientManager for the Cognito client.
Since we use the V2 code generator, ToolkitClientManager could actually be used to construct ToolkitTelemetryClient as well